src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -225,5 +227,206 @@ public static String getAttributeTableName(String clsName,
+	 * @return Set<GKInstance> -- All Human ReactionlikeEvents that are not used for manual inference.
+	 * @throws Exception-- Thrown by MySQLAdaptor
+	 */
+	public static Set<GKInstance> findHumanReactionsNotUsedForManualInference(MySQLAdaptor dba, List<String> skiplistDbIds) throws Exception {
+		Set<GKInstance> reactionsNotUsedForManualInference = new HashSet<>();
+		for (GKInstance event : findEventsNotUsedForManualInference(dba, skiplistDbIds)) {
+			// Filter for Human ReactionlikeEvents
+			if (event.getSchemClass().isa(ReactomeJavaConstants.ReactionlikeEvent)
+					&& isHumanDatabaseObject(event)) {
+
+				reactionsNotUsedForManualInference.add(event);
+			}
+		}
+		return reactionsNotUsedForManualInference;
+	}
+
+	/**
+	 * This method utilizes the InstanceUtilities method for finding all reaction participants. It also checks for the
+	 * 'activeUnit' participant in catalysts/regulations that can be flagged by certain QA checks. Generally speaking,
+	 * the 'activeUnit' instance should be found in the 'physicalEntity' of catalysts/regulations. Sometimes this is not
+	 * the case, when the wrong activeUnit is mistakenly checked in. This will be flagged by QA tests that call this method.
+	 * @param reaction GKInstance -- ReactionlikeEvent that will be checked for all participants.
+	 * @return Set<GKInstance> -- All distinct participants in the incoming ReactionlikeEvent.
+	 * @throws Exception -- Thrown by MySQLAdaptor.
+	 */
+	public static Set<GKInstance> getAllReactionParticipantsIncludingActiveUnits(GKInstance reaction) throws Exception {
+		Set<GKInstance> reactionPEs = new HashSet<>();
+		reactionPEs.addAll(InstanceUtilities.getReactionParticipants(reaction));
+		// Catalysts/Regulations also added to check for incorrect activeUnits.
+		reactionPEs.addAll(reaction.getAttributeValuesList(ReactomeJavaConstants.catalystActivity));
+		reactionPEs.addAll(reaction.getAttributeValuesList(ReactomeJavaConstants.regulatedBy));
+
+		Set<GKInstance> allReactionPEs = new HashSet<>();
+		for (GKInstance reactionPE : reactionPEs) {
+			allReactionPEs.add(reactionPE);
+			allReactionPEs.addAll(getPhysicalEntityContainedInstances(reactionPE));
+		}
+		return allReactionPEs;
+	}
+
+	/**
+	 * This method is wrapper for the InstanceUtilities getContainedInstances method.
+	 * @param reactionPE GKInstance -- PhysicalEntity that will be checked for contained instances.
+	 * @return Set<GKInstance> -- All distinct contained instances in the incoming PhysicalEntity.
+	 * @throws Exception -- Thrown by MySQLAdaptor.
+	 */
+	public static Set<GKInstance> getPhysicalEntityContainedInstances(GKInstance reactionPE) throws Exception {
+		return InstanceUtilities.getContainedInstances(
+				reactionPE,
+				ReactomeJavaConstants.hasMember,
+				ReactomeJavaConstants.hasCandidate,
+				ReactomeJavaConstants.hasComponent,
+				ReactomeJavaConstants.repeatedUnit,
+				ReactomeJavaConstants.activeUnit
+		);
```

Comment:
Querying activeUnit seems out of place here as all other attributes are part of Physical Entity sub-classes where activeUnit is part of CatalystActivity/Regulation.  If this can be queried in a separate method, that would be good because as it is, you have non-PE instances as part of the `reactionPE` set parameter.